src/content/docs/cheerpj3/11-guides/File-System-support.md

```diff
@@ -2,64 +2,96 @@
-| `/app/`   | An HTTP-based read-only filesystem, used to access JARs and data from your local server                   |
+| `/app/`   | An HTTP-based read-only filesystem used to access JARs and data from your local server                    |
 | `/files/` | An IndexedDB-based, persistent read-write file system                                                     |
-| `/lt/`    | Another HTTP-based read-only filesystem, pointing to the CheerpJ runtime                                  |
 | `/str/`   | A read-only filesystem to easily share JavaScript Strings or binary data (an `Uint8Array`) with Java code |
 
+![](/cheerpj2/assets/filesystem.png)
+
+> [!warning]Important
+> CheerpJ provides access to a virtualized filesystem, which does not correspond to the local computer. Accessing local files from the browser is forbidden due to security reasons.
+
 ## `/app/` mount point
 
 The /app/ mount point corresponds to a virtual read-only, HTTP-based filesystem. `/app/` is used to access JAR files and data from your local server.
 
-The `/app/` directory is virtual, and only exists inside of CheerpJ. It is required to distinguish files from the local server from runtime files and files stored in the browser database.
+This mount point is virtual and only exists inside of CheerpJ. It is required to distinguish between local server, runtime files and files stored in the browser database.
+
+The `/app/` mount point refers to the root of your web server. To have a clearer concept of the `/app/` mount point, let's assume that your web server is available at `http://127.0.0.1:8080/`:
 
-The `/app/` directory refers to the root of your web server. So, assuming that your web server is available at `http://127.0.0.1:8080/`, here are some example file mappings:
+- `/app/example.jar` would be the same as `http://127.0.0.1:8080/example.jar`
 
-- `/app/example.jar` → `http://127.0.0.1:8080/example.jar`
-- `/app/subdirectory/example.txt` → `http://127.0.0.1:8080/subdirectory/example.txt`
+- `/app/subdirectory/example.txt` would be the same as `http://127.0.0.1:8080/subdirectory/example.txt`
+
+Considering the examples above, to run a JAR with [`cheerpjRunJar`] and assuming it is stored in the root of the web server, it should be done as follows:
+
+```js
+cheerpjRunJar("/app/my_application_archive.jar");
+```
+
+> [!tip] Storing JAR files
+> The /app/ mount point is the most common location to store the application's JARs but this is not necessarily mandatory.
 
 ## `/files/` mount point
 
-The `/files/` mount point corresponds to a virtual read-write, IndexedDB-based filesystem. `/files/` is used to store persistent data on the browser client.
+The `/files/` mount point corresponds to a virtual read-write, IndexedDB-based filesystem and it is used to store persistent data in the browser client.
+
+Writing files into the `/files/` mount point is only possible from inside the Java application and specifying the root of the file path explicitly. For example:
```

Comment:
> and specifying the root of the file path explicitly

This is the case for any of  backend. There is no reason why `/files` should be special.